src/world/Action/EffectBuilder.h

```diff
@@ -13,30 +13,30 @@ namespace Sapphire::World::Action
 
     void heal( Entity::CharaPtr& effectTarget, Entity::CharaPtr& healingTarget, uint32_t amount,
                Common::ActionHitSeverityType severity = Common::ActionHitSeverityType::NormalHeal,
-               Common::ActionEffectResultFlag flag = Common::ActionEffectResultFlag::None);
+               Common::ActionEffectResultFlag flag = Common::ActionEffectResultFlag::None, uint64_t resultDelayMs = 600 );
```

Comment:
why is this hardcoded individually and differently in every function?

Reply:
just default testing value to use until real value is found.
However the 500 for status is very likely correct for single buff button.